### PR TITLE
First commit for 2022 robot. Contains SwerveBase code.

### PR DESCRIPTION
**Pull request by deslobodzian**

### CODE REVIEW

**Review by BenBernardCIS — APPROVED** [review]

**Comment by BenBernardCIS** [comment]
I'll use this branch to get Travis working, but you can go ahead and merge it after one more reviewer.

**Comment by greenermoose** [comment]
Great, thanks.

**Review by greenermoose — APPROVED** [review]
I trust this is all good!

File: .wpilib/wpilib_preferences.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "enableCppIntellisense": false,
+    "currentLanguage": "java",
+    "projectYear": "2021",
```
Comment (BenBernardCIS) [inline .wpilib/wpilib_preferences.json:4]: Should this be changed to 2022?

Comment (deslobodzian) [reply .wpilib/wpilib_preferences.json:4]: Yes, it should be 2022.

**Pull request merged**